src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -2798,35 +2903,6 @@ public static int Main()
             CreateCompilation(source).VerifyDiagnostics();
         }
 
-        // We used to return constant zero in unchecked conversion from a floating-point type to an integral type
-        // in a constant expression if the value being converted is not within the target type.
-        // The C# spec says that in this case the result of the conversion is an unspecified value of the destination type.
-        // Zero is a perfectly valid unspecified value, so that behavior was formally correct.
-        // But it did not agree with the behavior of the native C# compiler, that apparently returned a value that
-        // would resulted from a runtime conversion with normal CLR overflow behavior.
-        // To avoid breaking programs that might accidentally rely on that unspecified behavior
-        // we now match the native compiler behavior, and we are going to keep this behavior for compatibility.
-        [Fact, WorkItem(1020273, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1020273")]
-        public void Bug1020273()
```

Comment:
An internally reported "issue" such as [http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1020273](http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1020273) should not have taken priority over determinism (see [https://github.com/dotnet/roslyn/commit/c121bc53e4cadd98a1beda3c902e0521c833eaa8](https://github.com/dotnet/roslyn/commit/c121bc53e4cadd98a1beda3c902e0521c833eaa8)).  Now we have a user-reported issue about that determinism bug at [https://github.com/dotnet/roslyn/issues/37527](https://github.com/dotnet/roslyn/issues/37527), so I'm reverting that fix.

Reply:
I see that internally-reported issue was split from user reported http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1019366.